addons/web/static/src/views/calendar/calendar_renderer.scss

```diff
@@ -467,7 +469,7 @@
                     }
 
                     &:hover > .fc-day-number {
-                        --o-cw-color: #{$o-white};
+                        --o-cw-color: #fff;
```

Comment:
Here as well, let's choose a "safer' color (→ result is always `#ffffff`, but just to be sure).

Note that in the computation I replaced `$o-white`  with `$o-view-background-color` (the actual calendar's bg if I'm not wrong, to be tested).

```scss
$-bg: mix($o-view-background-color, $o-action, 50%);

--o-cw-color: #{color-contrast($-bg)};
--o-cw-bg: #{$-bg};
```

Reply:
In fact, using `color-contrast` with these parameters turns white into black. It seems that white doesn't provide enough contrast 🤔 .

| Community | Enterprise |
|--------|--------|
| ![Capture d’écran 2024-02-22 à 14 35 20](https://github.com/odoo/odoo/assets/80679690/3ac8b715-cb60-4c3f-822d-52f72b119b3e) | ![Capture d’écran 2024-02-22 à 14 34 49](https://github.com/odoo/odoo/assets/80679690/72369f7e-3f5f-4783-844b-694b75111884) |

Why not use the primary color? I think it would make more sense.

| Community | Enterprise |
|--------|--------|
| ![Capture d’écran 2024-02-22 à 14 37 37](https://github.com/odoo/odoo/assets/80679690/cc71af14-cbbe-4c9c-b97e-3c4d9483fe3d) | ![Capture d’écran 2024-02-22 à 14 39 02](https://github.com/odoo/odoo/assets/80679690/46ae6d10-0c27-44c1-ae44-fc14112c8089) |

What do you think @stefanorigano ?

Reply:
> with these parameters turns white into black. It seems that white doesn't provide enough contrast 🤔 .

The more contrasted color is (always?) the best one. Having said that,  if in specific circumstances you need to fine-tune the computation,  you can tweak the `$min-contrast-ratio` parameter when calling the function ([ref.](https://github.com/twbs/bootstrap/blob/1a6fdfae6be09b09eaced8f0e442ca6f7680a61e/scss/_functions.scss#L153)), eg

```scss
--o-cw-color: #{color-contrast($-bg, $min-contrast-ratio: 3)};
```

> Why not use the primary color

I think that primary should be use for the "selected" day, not the ":hovered" one.

Reply:
I did a few tests using `$min-contrast-ratio` but couldn't get anything to work well everywhere (Community, Enterprise, Darkmode).
So I tried decreasing the color of `$o-action` a little to increase the contrast.
```scss
 $-bg: mix($o-view-background-color, $o-action, 75%);

--o-cw-color: #{color-contrast($-bg)};
--o-cw-bg: #{$-bg};
```

| | Community | Enterprise | Darkmode |
|--------|--------|--------|--------|
| Before | ![Capture d’écran 2024-02-23 à 10 36 50](https://github.com/odoo/odoo/assets/80679690/3b14e26a-80de-4522-ae7a-785b0f047155) |  ![Capture d’écran 2024-02-23 à 10 36 40](https://github.com/odoo/odoo/assets/80679690/a27222ef-e344-4376-980b-3c5c5e1983e7) | ![Capture d’écran 2024-02-23 à 10 47 34](https://github.com/odoo/odoo/assets/80679690/a7cd6165-5487-482d-ac37-4dea2255aa65) |
| After | ![Capture d’écran 2024-02-23 à 10 35 22](https://github.com/odoo/odoo/assets/80679690/4ea8a24b-cc37-4030-bddb-459bcfe3da18) | ![Capture d’écran 2024-02-23 à 10 34 10](https://github.com/odoo/odoo/assets/80679690/8d5d1a4b-ff97-4e09-860f-7c28cd02daa5) | ![Capture d’écran 2024-02-23 à 10 48 50](https://github.com/odoo/odoo/assets/80679690/8eb4afd0-a6a6-4627-8228-f1882cbae7a0) |

What do you think @stefanorigano  ?